tools/frr-reload.py

```diff
@@ -260,6 +260,15 @@ def load_from_show_running(self, daemon):
                     not line):
                 continue
 
+            # some config lines appear for all daemons, which is an issue when computing the diff per-daemon
+            # so skip those if daemon is set and it's not zebra
```

Comment:
Err... I'm confused... the other daemons need the filters too?

Reply:
They do, and even with the patch above they will receive it when the config is applied to zebra; but there is no reason to have that config verbatim in all daemons, you will never want to delete it only in say ospfd or isisd. In integrated config this doesn't really matter, but when sing split config it makes more sense for this config to be assigned to a single daemon, so one doesn't have to apply the change to N different daemons/config files.

Reply:
Okay, but what if I delete some filters and then run to apply it only to isisd?

Reply:
how are you deleting the filters? our assumption is that you would do that either by modifying zebra.conf and calling frr-reload (in which case the patch would have no effect) or directly via vtysh (which would send the command to all daemons). In either case isisd would receive the changes and update its state, whatever you do or don't to isisd later should not matter. Basically you can ignore all that config part in the isisd.conf file, the assumption is that the source of truth for filters etc. is zebra. Ideally we wouldn't even render that configuration as part of `show running-config X` for `X != zebra`, and then this would not be needed

Reply:
I actually disagree with this change.  This is a downside of not having an integrated config.  If each daemon cannot start itself up after a crash because it's config file dos not have it's prefix-lists or route-maps or access-lists, we are going to be in a world of fun trying to figure this out.  How would a non-integrated-config restart of ripd work when it doesn't have it's route-map configuration?

Reply:
That's a very good point @donaldsharp. I dislike having the config replicated in each daemon but we will have to live with it for the time being.